docs/src/tutorials/symbolic_functions.md

```diff
@@ -372,15 +372,18 @@ the user's code. For these cases, Symbolics.jl allows for fully
 macro-free usage. For example:
 
 ```julia
+using Symbolics: Sym
+
 x = Num(Sym{Float64}(:x))
 y = Num(Sym{Float64}(:y))
-x+y^2.0 # isa Num
+x + y^2.0 # isa Num
 
-σ = Num(Variable{Symbolics.FnType{Tuple{Any},Real}}(:σ)) # left uncalled, since it is used as a function
-w = Num(Variable{Symbolics.FnType{Tuple{Any},Real}}(:w)) # unknown, left uncalled
-x = Num(Variable{Symbolics.FnType{Tuple{Any},Real}}(:x))(t)  # unknown, depends on `t`
+σ = Num(Variable{Symbolics.FnType{Tuple{Any}, Real}}(:σ)) # left uncalled, since it is used as a function
+w = Num(Variable{Symbolics.FnType{Tuple{Any}, Real}}(:w)) # unknown, left uncalled
+x = Num(Variable{Symbolics.FnType{Tuple{Any}, Real}}(:x))(t)  # unknown, depends on `t`
 y = Num(Variable(:y))   # unknown, no dependents
-z = Num(Variable{Symbolics.FnType{NTuple{3,Any},Real}}(:z))(t, α, x)  # unknown, multiple arguments
+# Line below throw an error since \alpha is not defined
+z = Num(Variable{Symbolics.FnType{NTuple{3, Any}, Real}}(:z))(t, α, x)  # unknown, multiple arguments
```

Comment:
just change alpha for something else?

Reply:
In other situation I would just replace `\alpha` by `y` and go on. But below we have line of code
`expr = \beta_{1} * x + y^{\alpha} + \sigma(3) * (z - t) - \beta_{2} * w(t - 1)`
so we should write `y^y` which is very different than `y^\alpha`. I don't what this example want to show, but I propose change it to `\alpha * y^2`. If this is ok, I will make PR or something like that. I'm still learning GitHub workflow.

Reply:
Then define an alpha?

Reply:
I correct the code in the simplest way that I found. I hope it is not bad correction, I don't get Symbolics.jl enough to be confident about my choices.